### DISCUSSION TITLE
Uncontrolled brightness decrease

### DISCUSSION THREAD

**Discussion by mra196** [Tyrrrz/LightBulb#229] [Q&A]
Hi

Thank you very much for this awesome freeware.

I've been encountering an issue with adjusting the brightness using shortcuts. When I decrease the brightness, it uncontrollably goes dark. Is there a workaround for this problem or do I have to change the brightness manually instead of with shortcut keys from now on (I never had this issue with the previous versions until my lightBulb got updated this morning). 

Cheers

**Comment by Tyrrrz** [comment]
That sounds like a bug, can you share your settings? At which base temperature/brightness does this happen?

Edit:
I think I found the bug. Can you try this build from CI (when it finishes)?
https://github.com/Tyrrrz/LightBulb/actions/runs/1556095349
